components/molecules/ProfileInfo.test.js

```diff
@@ -0,0 +1,20 @@
+import { render, screen } from '@testing-library/react'
+import '@testing-library/jest-dom/extend-expect'
+import { axe, toHaveNoViolations } from 'jest-axe'
+import ProfileInfo from './ProfileInfo'
+
+expect.extend(toHaveNoViolations)
+
+describe('ProfileInfo', () => {
+  it('renders ProfileInfo', () => {
+    render(<ProfileInfo />)
+    const text = screen.getByText('Ontario')
+    expect(text).toBeInTheDocument()
+  })
+
+  it('has no a11y violations', async () => {
+    const { container } = render(<ProfileInfo />)
```

Comment:
SUGESTING: If we move the render outside the `it` functions, into the `describe`, we only need to render the component once during the tests (speeding it up slightly).

Reply:
Great suggestion but this prevents us from rendering the component differently for future test cases. Since this component will take in props/data, we will most likely render it with different data and therefore, rendering the component differently.